pkg/utils/utils.go

```diff
@@ -16,7 +16,10 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/errors"
 )
 
-const maxFileSize = 1024 * 1024 * 1024 // 1GB
+const (
+	maxFileSize     = 1024 * 1024 * 1024 // 1GB
+	maxFileSizeInGB = float64(maxFileSize) / float64(1024*1024*1024)
```

Comment:
This seems a little silly
```suggestion
	maxFileSizeGb = 1
```